archivers/bzip3/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        kspalaiologos bzip3 1.2.1
+github.tarball_from releases
+name                bzip3
+version             1.2.1
+categories          archivers
+license             LGPL-3
+
+maintainers         {@sech1p gmail.com:sech1piam} openmainter
+
+description         A better and stronger spiritual successor to BZip2.
+long_description    {*}${description}
+
+homepage            https://github.com/kspalaiologos/bzip3
+
+platforms darwin
+
+checksums           rmd160  3940e049433a348ccc9fe4f0fcc3fe76cbae37ad \
+                    sha256  bba90d867f53efa4291de1df9c22da1578e1a93ca819e0fc68881da6fcc4149d \
+                    size    399621
+
+if {${os.arch} == "arm"} {
```

```suggestion
platform arm {
```
or that is mach better
```suggestion
if {${build_arch} ni [list i386 x86_64]} {
```